.github/ISSUE_TEMPLATE/early-design-review.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Early design review
+about: 'If you''d like the TAG to offer thoughts and guidance on an early-stage design
+  direction '
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Hello TAG!
+
+I'm requesting a TAG review of [feature name].
+
+[One paragraph summary of idea, ideally copy-pasted from Explainer introduction]
+
+  - Explainer¹ (minimally containing user needs and example code): [url]
+  - Security and Privacy self-review²: [url]
+  - GitHub repo (if you prefer feedback filed there): [url]
+  - Primary contacts (and their relationship to the specification): 
+      - [name] ([github username]), [organization/s] (repeat as necessary)
+  - Organization/project driving the design: [organization and/or project name] 
+  - External status/issue trackers for this feature (publicly visible, e.g. Chrome Status):
+
+Further details:
+
+  - [ ] I have reviewed the TAG's [API Design Principles](https://w3ctag.github.io/design-principles/)
+  - The group where the work on this design is being done (or is intended to be done in the future):
+  - Existing major pieces of multi-stakeholder review or discussion of this design: 
+  - Major unresolved issues or opposition with this design:
+  - This work is being funded by: 
+
+You should also know that...
+
+[please tell us anything you think is relevant to this review]
+
+We'd prefer the TAG provide feedback as (please delete all but the desired option):
+
+  🐛open issues in our GitHub repo for **each point of feedback**
+  🦋open a single issue in our GitHub repo **for the entire review**
+  💬leave review feedback as a **comment in this issue** and @-notify [github usernames]
+
+------------------------------------------------------------------------------------
+CAREFULLY READ AND DELETE CONTENT BELOW THIS LINE BEFORE SUBMITTING
+
+Please preview the issue and check that the links work before submitting.
+
+In particular:
+* if anything links to a URL which requires authentication (e.g. Google document), please make sure anyone with the link can access the document. We would prefer public documents though, since we work in the open.
+
+¹ For background, see our [explanation of how to write a good explainer](https://w3ctag.github.io/explainers). We recommend the explainer to be in [Markdown](https://github.github.com/gfm/).
+
+² Even for early-stage ideas, a Security and Privacy questionnaire helps us understand potential Security and Privacy issues and mitigations for your design, and can save us asking redundant questions. See https://www.w3.org/TR/security-privacy-questionnaire/.
```

Comment:
Downcase the second set of "security and privacy"

Reply:
Done.